pkg/actors/actors.go

```diff
@@ -1861,6 +1874,10 @@ func (a *actorsRuntime) storeReminder(ctx context.Context, store transactionalSt
 }
 
 func (a *actorsRuntime) GetReminder(ctx context.Context, req *GetReminderRequest) (*reminders.Reminder, error) {
+	if !a.config.HostedActorTypes.IsActorTypeHosted(req.ActorType) {
+		return nil, ErrReminderOpActorNotHosted
```

Comment:
Could add a test to cover this case?

Reply:
There are unit tests in pkg/http that cover this scenario, as well as E2E tests.

Additionally the unit tests in pkg/actors have been updated to make the actor type hosted, so the positive case is included.

I think that should be enough?